eng/mgmt/mgmtmetadata/compute_resource-manager.txt

```diff
@@ -3,12 +3,12 @@ AutoRest installed successfully.
 Commencing code generation
 Generating CSharp code
 Executing AutoRest command
-cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/compute/resource-manager/readme.md --csharp --version=v2 --reflect-api-versions --csharp-sdks-folder=D:\hylee-sdk\may\sdk
-2020-05-26 00:22:02 UTC
+cmd.exe /c autorest.cmd https://github.com/chetmaddula/azure-rest-api-specs/blob/master/specification/compute/resource-manager/readme.md --csharp --version=v2 --reflect-api-versions --csharp-sdks-folder=D:\GitSwagger\azure-sdk-for-net\sdk
+2020-06-05 18:37:50 UTC
 Azure-rest-api-specs repository information
-GitHub fork: Azure
+GitHub fork: chetmaddula
```

Comment:
Your Swagger PR is merged.  Please re-generate SDK from Azure master branch.

Reply:
Hi regenerated sdk from offical swagger